### PR TITLE
Make more fields VV and editable

### PR DESCRIPTION
**Pull request by DTanxxx**
Added some more VV and VVAccess.ReadWrite. However some may be inappropriate and may need to be changed back.

### CODE REVIEW

**Comment by DTanxxx** [comment]
This is for issue: [https://github.com/space-wizards/space-station-14/issues/2020]

File: Robust.Client/GameObjects/Components/Appearance/AppearanceComponent.cs [hunk 1]
```diff
@@ -24,8 +24,10 @@ public sealed class AppearanceComponent : SharedAppearanceComponent
 
         [Dependency] private readonly IReflectionManager _reflectionManager = default!;
 
+        [ViewVariables]
```
Comment (exp111) [inline Robust.Client/GameObjects/Components/Appearance/AppearanceComponent.cs]: This doesn't appear ingame, probably because it's static
grafik

Comment (PJB3005) [reply Robust.Client/GameObjects/Components/Appearance/AppearanceComponent.cs]: Yes this should probably be removed.

File: Robust.Client/Graphics/RSI/RSI.cs [hunk 2]
```diff
@@ -18,7 +18,7 @@ public sealed partial class RSI : IEnumerable<RSI.State>
         /// <summary>
         ///     The size of this RSI, width x height.
         /// </summary>
-        [ViewVariables]
+        [ViewVariables(VVAccess.ReadWrite)]
```
Comment (PJB3005) [inline Robust.Client/Graphics/RSI/RSI.cs]: You really should not make this writable with VV. You'll only break things.

If you *really* need to do advanced hacks like this I recommend you use CSI instead.

**Comment by DTanxxx** [comment]
Hi PJB3005,
Does my 'Applied feedback' commit look like what you were looking for? I have removed the VV and ReadWrite for the two variables. Please let me know if there are any more variables that should be made VV or ReadWrite.

**Pull request merged**